entc/gen/template/migrate/migrate.tmpl

```diff
@@ -38,6 +38,9 @@ var (
 	// WithFixture sets the foreign-key renaming option to the migration when upgrading
 	// ent from v0.1.0 (issue-#285). Defaults to true.
 	WithFixture = schema.WithFixture
+    // WithForeighKeys enables creating foreigh-key in ddl. In some case we are
+    // not allowed to use foreigh-key. Defaults to true
```

Comment:
This comment doc need to be expanded. 

The caveats need to be included in this comment + in ent website, because it affects the correctness of the data. See https://github.com/facebook/ent/issues/748#issuecomment-691623136

Reply:
has add documents in doc/md/migrate.md